lib/engine/game/g_18_eus/step/buy_sell_par_shares.rb

```diff
@@ -11,7 +11,7 @@ module G18EUS
       module Step
         class BuySellParShares < Engine::Step::BuySellParShares
           include Parrer
-          include BidboxAuction
+          include Engine::Game::G18EUS::Step::BidboxAuction
```

Comment:
Do you need the full scope? I think you just need `include BidboxAuction` since this class and the module have the same namespace.

Reply:
I thought it was clearer which one it was referring to - I can take it out.